### ISSUE TITLE
ServiceStack.Caching.Memcached assembly is not signed since version 5.5.0

### ISSUE THREAD

**Issue by gg3orgiev** [ServiceStack/Issues#689]
Hi, 

Up until version 5.4.0 of Service Stack, the **ServiceStack.Caching.Memcached.dll** provided with NuGet package was signed. Since version 5.5.0 of Service Stack, it is no longer strongly named and cannot be used into a signed project. This is the only assembly that is not signed from the whole NuGet bundle.

Is there a specific reason for which the assembly is no longer signed as it is currently blocking the use of the memcached integration for our project ?

Regards,
Georgi

**Comment by mythz** [comment]
Strange, all `net45` target builds should inherit the [same Directory.Build.props code-signing behavior](https://github.com/ServiceStack/ServiceStack/blob/632ec290cafcd2fcf51808018b46e56ad1670742/src/Directory.Build.props#L27-L32)

But it appears that changing:

```xml
<TargetFramework>net45</TargetFramework>
```

to 

```xml
<TargetFrameworks>net45</TargetFrameworks>
```

is required for it to apply the common **Directory.Build.props** behavior:

```xml
<PropertyGroup Condition=" '$(TargetFramework)' == 'net45'>
    <DefineConstants>$(DefineConstants);NET45</DefineConstants>
    <SignAssembly>True</SignAssembly>
    <DelaySign>False</DelaySign>
    <AssemblyOriginatorKeyFile>../servicestack.snk</AssemblyOriginatorKeyFile>
</PropertyGroup>
``` 

Anyway this should now be signed from v5.7.1+ that's now [available on MyGet](https://docs.servicestack.net/myget).

**Comment by gg3orgiev** [comment]
I can confirm the assembly is now signed in the new 5.7.1 NuGet packages. Thanks for the quick reaction.

Have a great day

**Comment by gg3orgiev** [comment]
Hi, 

Since version 6.0, this issue is present again. Only the Memcached assembly is not signed.

Regards,
Georgi

**Comment by mythz** [comment]
Must be same issue reappearing after upgrade to .NET v4.7.2, should be resolved from [this commit](https://github.com/ServiceStack/ServiceStack/commit/299f2ff52766aed1c5adffca578e4190ce5bc844).

This change is available from v6.4.1+ that's now [available on MyGet](https://docs.servicestack.net/myget).
